py3status/modules/nvidia_temp.py

```diff
@@ -1,17 +1,21 @@
 # -*- coding: utf-8 -*-
 """
-Display NVIDIA GPU temperature.
+Display NVIDIA GPU temperature(s).
 
 Configuration parameters:
     cache_timeout: how often we refresh this module in seconds (default 10)
-    format_prefix: a prefix for the output. (default 'GPU: ')
-    format_units: the temperature units. Will appear at the end. (default '°C')
-    temp_separator: the separator char between temperatures (only if more than
-        one GPU) (default '|')
+    format: display format for nvidia_temp (default 'GPU: {format_temp}')
+    format_temp: display format for gpu temperature(s) (default '{temp}°C')
+    icon_separator: display separator (only if more than one) (default '|')
+    msg_unavailable: display when unavailable (default 'GPU: N/A')
+
+Format placeholders:
+    {format_temp} display current gpu temperature(s)
+    {temp} display format for gpu temperature
```

Comment:
`{temp}` should be in its own section as an example see https://github.com/ultrabug/py3status/blob/master/py3status/modules/diskdata.py